loclist.kak

```diff
@@ -0,0 +1,147 @@
+## Location List
```

Comment:
Thanks so much for taking this up.
I've wanted to upstream [these](https://discuss.kakoune.com/t/single-command-for-grep-next-match-in-similar-buffers-lsp-make-find/1215) [two](https://discuss.kakoune.com/t/fricitionless-depth-first-search-with-grep-and-friends/2152) features for years.
Both are simple and usually obvious to the user.
The location stack is very useful and AFAICT no other editor has it, probably because they got stuck with more complex workflows.